src/meta/processors/job/BalancePlan.cpp

```diff
@@ -63,9 +63,8 @@ folly::Future<meta::cpp2::JobStatus> BalancePlan::invoke() {
           CHECK_EQ(j, buckets_[i].size() - 1);
           saveInStore();
           VLOG(4) << "BalancePlan::invoke finish";
-          auto result =
-              stopped ? meta::cpp2::JobStatus::STOPPED
-                      : failed_ ? meta::cpp2::JobStatus::FAILED : meta::cpp2::JobStatus::FINISHED;
+          // The task is either finished or failed here.
+          auto result = failed_ ? meta::cpp2::JobStatus::FAILED : meta::cpp2::JobStatus::FINISHED;
```

Comment:
@critical27  @panda-sheep  please help me to confirm when to set stoped. 
Now I keeped set stop logic in onError callback. I intend to change only here.